mifospay/src/main/res/layout/faq_list_child.xml

```diff
@@ -8,7 +8,10 @@
         android:id="@+id/faq_list_child"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_margin="16dp"
+        android:layout_marginLeft="@dimen/value_40dp"
+        android:layout_marginTop="@dimen/value_16dp"
+        android:layout_marginRight="@dimen/value_16sp"
```

Comment:
Change the `layout_marginRight` value also to density-independent pixels (**dp**)